lekce2/8.py

```diff
@@ -0,0 +1,11 @@
+# 8
+import sys
+import math
+
+print("Nacetli jsme prvni cislo: " + sys.argv[1])
+
+vysledek_zaokrouhlovani1 = math.floor(float(sys.argv[1])+0.5)
+print("zaokrouhleno " + str(vysledek_zaokrouhlovani1))
+
+# https://realpython.com/python-f-strings/
+print(f"zaokrouhleno {vysledek_zaokrouhlovani1}")
```

Comment:
Jen tak obecny koment, radsi bych ten soubor pojmenoval `08.py`, at je to rozumne serazene kdyz tam jdou cisla uloh pres desitku.